app/models/user.rb

```diff
@@ -2,4 +2,14 @@ class User < ActiveRecord::Base
   include Clearance::User
 
   has_many :itineraries
+
+  def testing_hound
+
+    true ? 'hi' : "bye"
+
+    puts "slkajfksdljflkdsaj fkldsaj fkljasdlk fjsadlk jfalksdj flkasdjf lkajsdkl fjsadlk fjlaksj d"
```

Comment:
Line is too long. [100/80]<br>Do not write to stdout. Use Rails' logger if you want to log.